pkg/resources/management.cattle.io/v3/project/validator.go

```diff
@@ -143,6 +151,48 @@ func (a *admitter) admitCommonCreateUpdate(oldProject, newProject *v3.Project) (
 	return admission.ResponseAllowed(), nil
 }
 
+func (a *admitter) validateContainerDefaultResourceLimit(limit *v3.ContainerResourceLimit) error {
+	if limit == nil {
+		return nil
+	}
+	fieldPath := projectSpecFieldPath.Child(containerLimitField)
+	requestsCPU, err := resource.ParseQuantity(limit.RequestsCPU)
+	if err != nil {
+		return field.Invalid(fieldPath, limit.RequestsCPU, "failed to parse container default requested CPU")
```

Comment:
I think it would be good in this case (and the others) to preserve the parsing error message as well:
```suggestion
		return field.Invalid(fieldPath, limit.RequestsCPU, fmt.Sprintf("failed to parse container default requested CPU: %s", err.Error()))
```

Reply:
Will adjust.